src/helpers/validateUrl.test.js

```diff
@@ -0,0 +1,95 @@
+import validateUrl from "./validateUrl";
+
+const validUrls = [
+  "http://foo.com/blah_blah",
+  "http://foo.com/blah_blah/",
+  "http://foo.com/blah_blah_(wikipedia)",
+  "http://foo.com/blah_blah_(wikipedia)_(again)",
+  "http://www.example.com/wpstyle/?p=364",
+  "https://www.example.com/foo/?bar=baz&inga=42&quux",
```

Comment:
Could we move the `https` example to the top of the array or add more `https` examples? To make it clearer that it's supported in the test. I was about to comment that we haven't got any examples of it until I found one buried in the sea of `http`! 😅

Reply:
Done :)